code/src/java/pcgen/gui2/csheet/CharacterSheetPanel.java

```diff
@@ -103,17 +103,13 @@ public CharacterSheetPanel()
 
 		// KAW TODO rewrite to use StatusWorker and PCGenTask for better progress display
 
-		refreshService = Executors.newSingleThreadExecutor(new ThreadFactory()
+		refreshService = Executors.newSingleThreadExecutor(r ->
 		{
-			@Override
-			public Thread newThread(Runnable r)
-			{
-				Thread thread = new Thread(r, "Character-Sheet-Refresher-Thread");
-				thread.setDaemon(true);
-				thread.setPriority(Thread.NORM_PRIORITY);
-				return thread;
-			}
-		});
+            Thread thread = new Thread(r, "Character-Sheet-Refresher-Thread");
```

Comment:
Another file with tabs vs spaces.  Getting very hesitant to do lots of your merges because it's creating inconsistency that other automated formatting will then try to redo and we get into thrashing lines that didn't really change.  Please convert to tabs.